velox/experimental/cudf/exec/ExpressionEvaluator.cpp

```diff
@@ -626,6 +642,22 @@ cudf::ast::expression const& AstContext::pushExprToTree(
     VELOX_CHECK_EQ(len, 3);
     auto node = CudfExpressionNode::create(expr);
     return addPrecomputeInstructionOnSide(0, 0, "split", "", node);
+  } else if (name == "hash_with_seed") {
+    // Only supports hash one value now.
+    VELOX_CHECK_EQ(len, 2);
+    auto literalExpr =
+        std::dynamic_pointer_cast<ConstantExpr>(expr->inputs()[0]);
+    VELOX_CHECK_NOT_NULL(literalExpr, "Expression is not a literal");
+    auto fieldExpr =
+        std::dynamic_pointer_cast<FieldReference>(expr->inputs()[1]);
+    VELOX_CHECK_NOT_NULL(fieldExpr, "Expression is not a field");
+
+    createLiteral(literalExpr->value(), scalars);
```

Comment:
since cudf needs a uint32 seed, there should be a check that the literal is either uint32 or convertible to it. I'd prefer this check to happen in the function object creation